modules/apps/transfer/types/packet_test.go

```diff
@@ -7,8 +7,10 @@ import (
 )
 
 const (
-	denom  = "transfer/gaiachannel/atom"
-	amount = uint64(100)
+	denom              = "transfer/gaiachannel/atom"
+	amount             = "100"
+	largeAmount        = "18446744073709551616"                                                           // one greater than largest uint64 (^uint64(0))
+	invalidLargeAmount = "115792089237316195423570985008687907853269984665640564039457584007913129639936" // 2^256
```

Comment:
is there a reason we've stopped the limit at `2^256`?

Reply:
That's what `sdk.Int` supports